ui/src/main/scripts/plugins/btm/ts/btm.ts

```diff
@@ -163,6 +169,24 @@ module BTM {
       $scope.faultChartHasData = txnFaultData.length > 0;
     };
 
+    $scope.addPropertyToFilter = function(pName, pValue, operator) {
```

Comment:
Don't see this called from here or the html, is it called from the filter sidebar?